### DISCUSSION TITLE
Separation of files storing on local

### DISCUSSION THREAD

**Discussion by dimitrov-adrian** [directus/directus#3887] [Feature Requests]
Currently every file that is uploaded on local file system is going to one uploads/ directory, when applying transformations (system and custom) these files are multiplied. I am thinking, if it could be good to have some file separation at least with the first 1-2 chars, so when have a lot of files simple `ls` not to take ages.

Example:
`./uploads/a8a8192a-fbb9-4ad1-ae01-e927552095ce.jpeg` -> `./uploads/a8/a8192a-fbb9-4ad1-ae01-e927552095ce.jpeg`

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
